src/ray/util/logging.cc

```diff
@@ -312,14 +311,42 @@ void RayLog::InitLogFormat() {
   }
 }
 
+/*static*/ std::string RayLog::GetLogOutputFilename(const std::string &log_dir,
+                                                    const std::string &log_file,
+                                                    const std::string &app_name) {
+  if (!log_file.empty()) {
+    return log_file;
+  }
+  if (!log_dir.empty()) {
+#ifdef _WIN32
+    int pid = _getpid();
+#else
+    pid_t pid = getpid();
+#endif
+
+    return JoinPaths(log_dir, absl::StrFormat("%s_%d.log", app_name, pid));
+  }
+  return "";
+}
+
 void RayLog::StartRayLog(const std::string &app_name,
                          RayLogLevel severity_threshold,
-                         const std::string &log_dir) {
+                         const std::string &log_dir,
+                         const std::string &log_filepath,
+                         size_t log_rotation_max_size,
+                         size_t log_rotation_file_num) {
```

Comment:
I don't see where it's used

Reply:
I haven't finished implementation.

Reply:
Passing argument is not ideal in a way, since you need to define a priority between env vs passed var.

Reply:
Yes, updated.